src/pages/trade-types/multiplier/__basket_indices.tsx

```diff
@@ -0,0 +1,199 @@
+    border-left: 1px solid var(--color-grey-22);
+    border-right: 1px solid var(--color-grey-22);
+    width: 100%;
+    padding: 2.4rem 0.7rem;
+    grid-row-gap: 1.6rem;
+    grid-template-columns: repeat(3, 1fr);
+    @media ${device.tabletL} {
+        grid-template-columns: repeat(2, 1fr);
+
+        img {
+            width: 16px;
+            height: 16px;
+            margin-right: 4px;
+        }
+        ${Text} {
+            font-size: 12px;
+            line-height: 1.5;
+        }
+    }
+    @media ${device.tabletS} {
+        grid-template-columns: repeat(1, 1fr);
+    }
+`
+
+const Title = styled(Text)`
+    text-align: center;
+    @media ${device.tabletL} {
+        font-weight: 600;
+    }
+`
+
+const DetailsContainer = styled(Flex)`
+    flex-direction: column;
+
+    ${Text} {
+        font-size: 1.4rem;
+        margin-top: 1.6rem;
+        @media ${device.tabletL} {
+            margin-top: 1rem;
+        }
+    }
+`
+const AvailablePlatformsWrapper = styled(Flex)`
+    border-bottom: 1px solid var(--color-grey-21);
+    padding-bottom: 40px;
+    @media ${device.tabletL} {
+        margin-bottom: 24px;
+        padding-bottom: 16px;
+    }
+`
+
+const BasketFXDetails = () => (
+    <DetailsContainer>
+        <Text>
+            <Localize
+                translate_text="<0>AUD Basket</0> measures the value of the Australian dollar against a basket of five global currencies (USD, EUR, GBP, JPY, CAD), each weighted by 20%."
+                components={[<strong key={0} />]}
+            />
+        </Text>
+        <Text>
+            <Localize
+                translate_text="<0>EUR Basket</0> measures the value of the Euro against a basket of five global currencies (USD, AUD, GBP, JPY, CAD), each weighted by 20%."
+                components={[<strong key={0} />]}
+            />
+        </Text>
+        <Text>
+            <Localize
+                translate_text="<0>GBP Basket</0> measures the value of the British pound against a basket of five global currencies (USD, EUR, AUD, JPY, CAD), each weighted by 20%."
+                components={[<strong key={0} />]}
+            />
+        </Text>
+        <Text>
+            <Localize
+                translate_text="<0>USD Basket</0> measures the value of the US dollar against a basket of five global currencies (EUR, GBP, JPY, CAD, AUD), each weighted by 20%."
+                components={[<strong key={0} />]}
+            />
+        </Text>
+    </DetailsContainer>
+)
+const BasketCommoditiesDetails = () => (
+    <DetailsContainer>
+        <Text>
+            <Localize
+                translate_text="<0>Gold Basket</0> measures the value of gold against a basket of five global currencies (EUR, GBP, JPY, AUD, USD), each weighted by 20%."
+                components={[<strong key={0} />]}
+            />
+        </Text>
+    </DetailsContainer>
+)
+
+const BasketIndices = () => {
+    const { is_eu_country } = React.useContext(DerivStore)
```

Comment:
Let's use the `is_eu` from `getCountryRule`

Reply:
Removed the 2 lines of code as mentioned above so no need for this line.